.github/scripts/test_issue_2246.py

```
@@ -0,0 +1,893 @@
+                    f"命中根因路径日志 {diag_counter['path_hits']} 次",
+                    flush=True,
+                )
+    finally:
+        stop_event.set()
+        if target_sock is not None:
+            close_session_socket(target_sock, args.close_mode)
+        for thread in threads:
+            thread.join(timeout=2.0)
+
+    if hit_line[0]:
+        print("[结果] 已命中目标故障窗口（测试失败）。", flush=True)
```

Ruff (0.15.6)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)